OutOfSchool/OutOfSchool.IdentityServer/Controllers/AccountController.cs

```diff
@@ -196,6 +196,33 @@ public async Task<IActionResult> SendConfirmEmail()
         return Ok();
     }
 
+    [HttpGet]
+    public async Task<IActionResult> ReSendEmailConfirmation()
+    {
+        var user = await userManager.FindByEmailAsync(User.Identity.Name);
+
+        var token = await userManager.GenerateEmailConfirmationTokenAsync(user);
+        var returnUrl = "Login";
```

Comment:
`var returnUrl = "Login"` sounds like a constant, that shoud be set on higher level (not a local method variable)